src/Engine.php

```diff
@@ -80,7 +80,7 @@ public function remove($searchableEntities): array
         /** @var SearchableEntity $entity */
         foreach ($searchableEntities as $entity) {
             $searchableArray = $entity->getSearchableArray();
-            if (null === $searchableArray || 0 === \count($searchableArray)) {
+            if (null == $searchableArray || 0 == \count($searchableArray)) {
```

Comment:
Why this change? :)

Reply:
This change came from PHPstan stating that this comparison is always `null`. I prepared an example in the PHPstan playground: https://phpstan.org/r/07c0aa63-1a31-4a24-b6af-c14dffa92dc5